api/tests/test-mount.js

```diff
@@ -68,6 +68,71 @@ o.spec("mount", function() {
 		}, FRAME_BUDGET)
 	})
 
+	o("redraws several mount points on events", function(done) {
+		var onupdate0 = o.spy()
+		var oninit0   = o.spy()
+		var onclick0  = o.spy()
+		var onupdate1 = o.spy()
+		var oninit1   = o.spy()
+		var onclick1  = o.spy()
+
+		var e = $window.document.createEvent("MouseEvents")
```

Comment:
I thought it was `MouseEvent`, not `MouseEvents`. I may be wrong, since this is right off the top of my head.